Notification Service/NotificationService.swift

```diff
@@ -81,7 +81,7 @@ class NotificationService: UNNotificationServiceExtension {
                 let config = ParseClientConfiguration { configuration in
                     configuration.applicationGroupIdentifier = Config.shared.environment.groupId
                     // Allow parse to access the data from the main app bundle.
-                    configuration.containingApplicationBundleIdentifier = "com.Jibber-Inc.iOS"
+                    configuration.containingApplicationBundleIdentifier = Config.shared.environment.bundleId
```

Comment:
This should fix our notification issues, but you should test it with your device.